sbatch_scripts/many_small_jobs.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#SBATCH --account=windse
+#SBATCH --time=48:00:00
+#SBATCH --nodes=1
+#SBATCH --job-name=many_small_jobs
+
+ 
+# Load and activate your conda environment
+module purge
+module load conda
+source activate windse
+ 
+# This stops the solver from filling all 36 cores
+# Not using this may significantly slow down execution (100 times slower on Eagle)
+export OMP_NUM_THREADS=1
+
+
+# Number of times all processors run a job = # of jobs/ # of cores per node
+N_LOOP=2
+# Run 1 job per task
+N_JOB_PER_LOOP=$SLURM_NTASKS
+
+# Where to store Logs
+mkdir -p LOGS
+
+
+# Run many small jobs
+for((j=0;j<$N_LOOP;j++))
+do
+
+  for((i=0;i<$N_JOB_PER_LOOP;i++))
+  do
+
+    jobid=$((i + j*N_JOB_PER_LOOP))
+    windse run test.yaml -p 'general':'name':test_$jobid > LOGS/log$jobid &
```

Comment:
Since this is kind of a tutorial/how-to document, it might be good to include a little more explanation here, something like:
```bash
    # Construct a unique job ID
    jobid=$((i + j*N_JOB_PER_LOOP))

    # Create a unique name for this job and its output
    jobname=$(printf "test_job_%03d" $jobid)
    logname=$(printf "log_%03d" $jobid)

    # Run the job
    windse run test.yaml -p 'general':'name':$jobname > LOGS/$logname &

```
this also allows us to format the names with leading zeros which makes command-line display of big parameter sweeps a little friendlier.

Reply:
Good point, I added the proposed output formatting it in the edited PR.